LockboxXCUITests/LockBoxScreenGraph.swift

```diff
@@ -201,102 +213,57 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
 }
 
 extension BaseTestCase {
-
-   func disconnectAndConnectAccount() {
-        navigator.performAction(Action.DisconnectFirefoxLockbox)
-        // And, connect it again
-        waitforExistence(app.buttons["getStarted.button"])
-        app.buttons["getStarted.button"].tap()
-        userState.fxaUsername =  emailTestAccountLogins
+    func loginFxAccount() {
         userState.fxaPassword = passwordTestAccountLogins
+        userState.fxaUsername = emailTestAccountLogins
+        navigator.goto(Screen.FxASigninScreenEmail)
+        waitforExistence(app.navigationBars["Lockbox.FxAView"])
         waitforExistence(app.webViews.textFields["Email"], timeout: 10)
-        navigator.nowAt(Screen.FxASigninScreenEmail)
         navigator.performAction(Action.FxATypeEmail)
+        waitforExistence(app.webViews.secureTextFields["Password"])
         navigator.performAction(Action.FxATypePassword)
     }
 
-    func checkIfAccountIsVerified() {
-        if (app.webViews.staticTexts["Confirm this sign-in"].exists) {
-            let group = DispatchGroup()
-            group.enter()
-            DispatchQueue.global(qos: .userInitiated).async { [weak self] in
-                self?.verifyAccount() {
-                    group.leave()
-                }
+    func skipAutofillConfiguration() {
+        if #available(iOS 12.0, *) {
+            //If autofill is set this option will not appear
+            sleep(5)
```

Comment:
This is the only thing I can't fix, depending on the autofill setting the option will be available or not. I can't use a `waitForExistence` it will fail if the option is set previously. I know is not a good solution but don't know what else I could do...